src/utils/service/service.js

```diff
@@ -255,7 +255,7 @@ class Service {
    * @return {Void}
    */
   responseTransform(response) {
-    return JSON.parse(response);
+    return JSON.parse(!response ? '{}' : response);
```

Comment:
I'm not sure returning an empty object here is the right approach, as if there is no response body it could make it seem as though there is e.g. with an empty response body returning an empty object from `responseTransform` would evaluate to `truthy`.

Would it be better to allow `responseTransform` to return `undefined` when there is no response body?

Reply:
I think we need to do this in order for the response to be correctly parsed?

@adamgeorgeson might be able to shed some light, I picked this up quickly before the dentist so wasn't fully clear on the underlying issue!

Reply:
@ianoxley That seems fair. Will pick it up.

Reply:
Parking this for now, as returning `undefined` in `responseTransform` ends up requiring us to capture `response.data` being `undefined` in `handleSuccess` which would need to be handled correctly in order to ensure we trigger callbacks. Will have a catchup with @ianoxley and @andrewjtait in the morning.

Reply:
This has been addressed in 6c77470 @andrewjtait @ianoxley